.github/actions/release-maven/action.yaml

```diff
@@ -5,9 +5,6 @@ inputs:
   module:
     description: 'Module to build'
     required: true
-  release:
-    description: 'Release?'
-    default: 'false'
   release-version:
     description: 'Release version'
     required: false
```

Comment:
This must be `required`, because its needed in line 51

Reply:
Done